config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get '/posts', to: 'posts#index'
+  get '/posts/:id', to: 'posts#show'
+  get '/users', to: 'users#index'
+  get '/users/:id', to: 'users#show'
```

Comment:
Great job @edubew 
### **OPTIONAL**
- [ ] Please make use of the resource routes.

```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
I will do that🙂. Thank you @CollinsTatang for your  feedback